src/actions/requests.rs

```diff
@@ -112,6 +112,7 @@ impl RequestAction for Symbols {
 
         Ok(symbols
             .into_iter()
+            .filter(|s| !s.name.is_empty())
```

Comment:
Could we add a `HACK` stating that this is working around VSCode symbol handling logic?

Reply:
Good idea, especially since LSP is ambiguous.